hepcrawl/parsers/jats.py

```diff
@@ -0,0 +1,529 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of hepcrawl.
+# Copyright (C) 2015, 2016, 2017 CERN.
+#
+# hepcrawl is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Extractrs for various metadata formats"""
+
+from __future__ import absolute_import, division, print_function
+
+import itertools
+
+import six
+
+from inspire_schemas.api import LiteratureBuilder, ReferenceBuilder
+from inspire_schemas.utils import build_pubnote, split_page_artid
+from inspire_utils.date import PartialDate
+from inspire_utils.helpers import maybe_int, remove_tags
+
+from lxml import etree  # needed until a new release of parsel is made
+
+from ..utils import best_match, get_node
+
+
+class JatsParser(object):
+    """Parser for the JATS format.
+
+    It can be used directly by invoking the :func:`parse` method, or be
+    subclassed to customize its behavior.
+
+    Args:
+        jats_record (Union[str, scrapy.selector.Selector]): the record in JATS format to parse.
+        source (Optional[str]): if provided, sets the ``source`` everywhere in
+            the record. Otherwise, the source is extracted from the JATS metadata.
+
+    Returns:
+        object: the same record in the Inspire Literature schema.
```

Comment:
This return is wrong right? I guess that it's for the `__init__` method, that will return an instance of `JatsParser`, that does not represent a record if I'm not mistaken. Probably it was moved from the parse method?

Reply:
It does explain lack of `Returns:` in `parse` docstring, I can move it